packages/bsky/tests/auto-moderator/labeler.test.ts

```diff
@@ -94,6 +100,28 @@ describe('labeler', () => {
       val: 'test-label',
       neg: false,
     })
+
+    // Verify that along with applying the labels, we are also leaving trace of the label as moderation event
+    // Temporarily assign an instance of moderation service to the autoMod so that we can validate label event
+    const modSrvc = autoMod.services.moderation(ctx.db)
+    const { events } = await modSrvc.getEvents({
+      includeAllUserRecords: false,
+      subject: uri.toString(),
+      limit: 10,
+      types: [],
+    })
+    expect(events.length).toBe(1)
+    expect(events[0]).toMatchObject({
+      action: 'com.atproto.admin.defs#modEventLabel',
+      subjectUri: uri.toString(),
+      createLabelVals: 'test-label',
+      negateLabelVals: null,
+      comment: `[AutoModerator]: Applying labels`,
+      createdBy: labelerDid,
+    })
+
+    // Cleanup the temporary assignment, knowing that by default, moderation service is not available
+    autoMod.services.moderation = undefined
```

Comment:
feels a tiny bit hacky, maybe doing this through before and after hook would look nicer? lmk if anyone feels strongly about this.